x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/rollup_wizard/steps/step_date_histogram.js

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@kbn/i18n/react';
+import moment from 'moment-timezone';
+
+import {
+  EuiDescribedFormGroup,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiSelect,
+  EuiSpacer,
+  EuiTitle,
+  EuiSuperSelect,
+  EuiText,
+} from '@elastic/eui';
+
+import { CALENDAR_INTERVAL_OPTIONS } from '../constants';
+
+import { StepError } from './components';
+
+const timeZoneOptions = moment.tz.names().map((name) => ({
+  value: name,
+  text: name,
+}));
+
+const calendarIntervalOptions = [
+  {
+    value: CALENDAR_INTERVAL_OPTIONS.minute,
+    text: i18n.translate(
+      'xpack.indexLifecycleMgmt.rollup.create.stepDateHistogram.fieldIntervalField.minuteOptionLabel',
+      { defaultMessage: 'Every minute' }
```

Comment:
"Every" makes it seem like something is happening once per minute, rather than reducing the granularity of the data. For everything but minute, "Hourly summaries, Daily summaries, etc." might work.  Maybe 60-second summaries?